ui/src/main/js/view/widgets/dialog/style.less

```diff
@@ -1,13 +1,16 @@
 .cbwf-dialog {
   border-radius: @cbwf-border-radius;
   border: solid 1px @dark-border-color;
+  border: solid 1px var(--panel-border-color);
   background-color: @light-body-background-color;
+  background-color: var(--background);
   box-shadow:rgba(0,0,0,.25) 0 0 0 999em, rgba(0,0,0,.25) 0 0 0 1px, rgba(0,0,0,.33) 0 3px 10px;
 
   .header {
     border-radius-topleft: @cbwf-border-radius;
     border-radius-topright: @cbwf-border-radius;
     background-color: @light-header-background-color;
+    background-color: var(--bigtable-header-color);
```

Comment:
Ditto

Reply:
This one should probably be named better if it's to be used with more than just bigtable

Reply:
I'm probably going to be restyling the tables on Jenkins in a few weeks. Maybe we can discuss the CSS vars API for tables then.

Reply:
sure for now, I think we can add this, and dark-theme will support it, if it disappears from Jenkins it will just fallback to what was there previously

Once the API has been sorted we can adjust here and dark-theme can support both for a little bit

Reply:
Sounds like a plan!

Reply:
I would only then suggest leaving a comment indicating that this variable is brittle.

Reply:
```suggestion
    // This variable may be renamed on Jenkins core in the future and stop working
    background-color: var(--bigtable-header-color);
```

Reply:
not sure if it's needed really, there's a fallback added.